test/csuite/timestamp_abort/main.c

```diff
@@ -178,6 +190,7 @@ thread_ts_run(void *arg)
                   "oldest_timestamp=%s,stable_timestamp=%s", ts_string, ts_string));
```

Comment:
Since reverting this will only be removing the "don't go backward" part above, I would like to see the other change I suggested in this ticket go in here. That is, adding a case here where we set the oldest and stable timestamps in 2 separate API calls. Otherwise that suggestion will get lost in later tickets.